<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>701</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1040</width>
    <height>701</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1040</width>
    <height>701</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1040</width>
    <height>701</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>157</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>uisc/blank_2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>157</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>uisc/blank_1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>157</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>uisc/blank_3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>157</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>uisc/blank_4.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/1.png</normaloff>uisc/1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>140</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/2.png</normaloff>uisc/2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/3.png</normaloff>uisc/3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/4.png</normaloff>uisc/4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>1155</width>
     <height>155</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>310</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/5.png</normaloff>uisc/5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/6.png</normaloff>uisc/6.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/7.png</normaloff>uisc/7.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/8.png</normaloff>uisc/8.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>440</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/9.png</normaloff>uisc/9.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>440</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/0.png</normaloff>uisc/0.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_arrow">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>570</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/arrow.png</normaloff>uisc/arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>1555</width>
     <height>1555</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>570</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>uisc/검색.png</normaloff>uisc/검색.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_maintitle">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>37</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>uisc/주차감면신청.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_subtitle">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>100</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>uisc/주차하신_차량의_차량번호_뒷자리(4자리)를_입력하세요.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="box_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a아시아고딕L</family>
     <pointsize>80</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="box_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a아시아고딕L</family>
     <pointsize>80</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="box_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a아시아고딕L</family>
     <pointsize>80</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="box_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a아시아고딕L</family>
     <pointsize>80</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
